service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt

```diff
@@ -456,15 +456,15 @@ FROM missing_planned_departures WHERE realtime.id = missing_planned_departures.i
         .bind("now", now)
         .execute()
 
-    val defaultDepartureTime = now.minusDays(1).withTime(LocalTime.of(18, 0))
+    val defaultDepartureTime = now.minusDays(1).withTime(LocalTime.of(20, 0))
     createUpdate(
             // language=SQL
             """
 UPDATE staff_attendance_realtime a
 SET departed = :defaultDepartureTime
```

Comment:
Instead of basically duplicating the updates you could also replace this with `SET departed = CASE WHEN arrived < :defaultDepartureTime THEN :defaultDepartureTime ELSE :startOfDay END` which would also be simpler.